### PR TITLE
Postman Sort in C#

### PR DESCRIPTION
**Pull request by ritish099**
Fixes # #2251

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Created a Folder Postman_Sort
- Added a file Postman_Sort.cs

#### Languages Used:
- C#

#### Files Added:
- Postman_Sort.cs

Thanks!

### CODE REVIEW

**Comment by ritish099** [comment]
@jainaman224 check it out sir

**Comment by ritish099** [comment]
@MastersAbh @asha15 @SakshiShreya  Pls review it. Pending from 10 days

File: Postman_Sort/Postman_Sort.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                while (t > 0)
+                {
+                    p.c++; //Counting the no of elements of a digit
+                    t = t / 10; 
+                }
+                if (p.maxdigits < p.c)
+                    p.maxdigits = p.c; //Storing the length of longest digit
+                p.c = 0;
+            }
+            for(i=1;i<p.maxdigits;i++)
```
Comment (MastersAbh) [inline Postman_Sort/Postman_Sort.cs]: space around operators

File: Postman_Sort/Postman_Sort.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
+﻿//Program to Sort the array using Postman Sort
+
+using System;
+namespace Declaring_Method
+{
+    public class Sort
```
Comment (MastersAbh) [inline Postman_Sort/Postman_Sort.cs]: rename class to Postman_Sort

**Comment by MastersAbh** [comment]
Please squash you commits

**Comment by ritish099** [comment]
@MastersAbh check it out

**Comment by ritish099** [comment]
and also check my one more PR that is pending

**Comment by ritish099** [comment]
Link to that PR [Another PR](https://github.com/jainaman224/Algo_Ds_Notes/pull/2626)

**Pull request merged**